### PR TITLE
Show one style

### PR DESCRIPTION
**Pull request by jackeymason**
Modified files based on previous pull request feedback.

### CODE REVIEW

File: src/main/java/com/launchacademy/reviews/controllers/PizzaStylesApiV1Controller.java
Comment (safang99) [inline src/main/java/com/launchacademy/reviews/controllers/PizzaStylesApiV1Controller.java:36]: Nice update here!

File: src/main/frontend/components/PizzaStyleShow.js
Comment (safang99) [inline src/main/frontend/components/PizzaStyleShow.js:46]: Looks like some extra stuff about delete and edit may have made it's way into this PR and should be removed to keep it in scope

File: src/main/java/com/launchacademy/reviews/controllers/PizzaStylesApiV1Controller.java
Comment (safang99) [inline src/main/java/com/launchacademy/reviews/controllers/PizzaStylesApiV1Controller.java:51]: Another delete reference to remove

**Review by safang99 — CHANGES_REQUESTED** [review]
Looks like another feature branch may have been merged in here.  I would remove all the extra features that aren't needed for this current story of getting the show page up

File: src/main/java/com/launchacademy/reviews/controllers/PizzaStylesApiV1Controller.java
Comment (jackeymason) [reply src/main/java/com/launchacademy/reviews/controllers/PizzaStylesApiV1Controller.java]: I had to keep working and while waiting for a PR approval, so I branched off of this branch. Somehow those changes bled upstream. One has to assume I wasn't working in the branch I thought I was working in.

**Comment by jackeymason** [comment]
Pull request. Helen made heroic efforts to detangle merged branches.

**Pull request merged**
